source/compliance/nist/configuration-assessment.rst

```diff
@@ -46,20 +46,11 @@ This use case shows how Wazuh helps meet the **CM-6 Configuration settings** req
       :align: center
       :width: 80%
 
-
-#. Select **CIS benchmark for Ubuntu Linux 22.04**.
-
-   .. thumbnail:: /images/compliance/nist/select-cis-benchmark.png    
-      :title: Select CIS benchmark
-      :alt: Select CIS benchmark
-      :align: center
-      :width: 80%
```

Comment:
**thought**: depending on the policies for the agent, could be needed to select it.

Reply:
Solved